src/TextInputWithIcon/index.jsx

```diff
@@ -140,6 +140,14 @@ class TextInputWithIcon extends Component
          *  Display as read-only
          */
         isReadOnly            : PropTypes.bool,
+        /**
+         *  Display as read-only for TextInput
+         */
+        isReadOnlyInput       : PropTypes.bool,
+        /**
+         *  Display as read-only for IconButton
+         */
+        isReadOnlyBtn         : PropTypes.bool,
```

Comment:
Generally we don’t abbreviate `Button` 👉 `Btn`.  So let’s change this to `isReadOnlyButton` to be consistent with the naming used in other components.

Reply:
Sure 👍

Reply:
Done, Checking...

Reply:
Passed.